js-oop/game/game.js

```diff
@@ -0,0 +1,21 @@
+function GameObject(imageUrl) {
+    this.imageUrl = imageUrl;
```

Comment:
We don't have imageUrl field, we have image and it must be initialized with new Image()